src/applications/hca/tests/reducers.unit.spec.js

```diff
@@ -0,0 +1,141 @@
+// import sinon from 'sinon';
+import { expect } from 'chai';
```

Comment:
thanks for adding this

Reply:
heh and thanks for making me notice line 1 which should be removed!